gui/hud.gui

```diff
@@ -2991,6 +2991,743 @@ widget = {
+						texture = "gfx/interface/hud/hud_infusion_glow_purple.dds"
+
+						blockoverride "animation_trigger_1" {
+							name = "lvl3_fire"
+							trigger_on_create = yes
+						}
+
+						blockoverride "animation_trigger_2" {}
+						blockoverride "lvl_3" {}
+					}
+
+					icon = {
+						parentanchor = center
+						position = { -30 0 }
+						size = { 30 30 }
+						texture = "gfx/interface/colors/white.dds"
+						using = Color_Red
+
+
+						modify_texture = {
+							name = "mask_1"
+							texture = "gfx/interface/component_masks/mask_glow.dds"
+							blend_mode = alphamultiply
+							rotate_uv = 1
+						}
+
+						modify_texture = {
+							name = "mask_2"
+							texture = "gfx/interface/component_masks/mask_glow.dds"
+							blend_mode = alphamultiply
+							rotate_uv = -1
+						}
+
+						modify_texture = {
+							name = "mask_3"
+							texture = "gfx/interface/component_masks/mask_clouds.dds"
+							blend_mode = alphamultiply
+							alpha = 0.1
+						}
+
+						modify_texture = {
+							name = "mask_4"
+							texture = "gfx/interface/component_masks/mask_circle.dds"
+							blend_mode = mask
+						}
+
+						modify_texture = {
+							name = "mask_3"
+							texture = "gfx/interface/colors/gold.dds"
+							blend_mode = add
+						}
+
+						state = {
+							name = "stress_level_increase"
+							next = b
+
+							using = Animation_Curve_Default
+							scale = 1
+							alpha = 0
+						}
+
+						state = {
+							name = b
+							next = c
+
+							duration = 0.6
+							bezier = { 0.5 0 1 0.5 }
+
+							scale = 5
+							alpha = 0.7
+						}
+
+						state = {
+							name = c
+							duration = 0.3
+							alpha = 0
+							scale = 6
+						}
+
+						state = {
+							name = rotate_a
+							next = rotate_b
+							trigger_on_create = yes
+
+							modify_texture = {
+								name = "mask_1"
+								rotate_uv = 1
+							}
+						}
+
+						state = {
+							name = rotate_a
+							next = rotate_b
+
+							duration = 1
+
+							modify_texture = {
+								name = "mask_1"
+								rotate_uv = 180
+							}
+						}
+
+						state = {
+							name = rotate_a_2
+							next = rotate_b_2
+							trigger_on_create = yes
+
+							modify_texture = {
+								name = "mask_2"
+								rotate_uv = -1
+							}
+						}
+
+						state = {
+							name = rotate_a_2
+							next = rotate_b_2
+
+							duration = 1
+
+							modify_texture = {
+								name = "mask_2"
+								rotate_uv = -180
+							}
+						}
+
+						state = {
+							name = rotate_a_3
+							next = rotate_b_3
+							trigger_on_create = yes
+
+							modify_texture = {
+								name = "mask_3"
+								rotate_uv = 0
+								alpha = 0.1
+							}
+						}
+
+						state = {
+							name = rotate_a_3
+							next = rotate_b_3
+
+							duration = 1
+
+							modify_texture = {
+								name = "mask_3"
+								rotate_uv = 40
+								alpha = 0.8
+							}
+						}
```

Comment:
Do you know what these things are doing?

Reply:
That's for the animations on either the bar or the particles coming of it

Reply:
> That's for the animations on either the bar or the particles coming of it

I removed all these states on one of these 2 images. Try to guess which one.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123148516-61139b80-d471-11eb-9cce-d4840349a968.png)
![image](https://user-images.githubusercontent.com/46384992/123148634-856f7800-d471-11eb-8719-64e7cfeadda2.png)

</details>

Reply:
Bottom